src/library/playlisttablemodel.cpp

```diff
@@ -170,7 +175,16 @@ void PlaylistTableModel::setTableModel(int playlistId) {
             LIBRARYTABLE_ID,
             columns,
             m_pTrackCollectionManager->internalCollection()->getTrackSource());
-    setSearch("");
+
+    // Restore search text
+    QString prevSearch("");
```

Comment:
QString is empty by default. 
```suggestion
    QString prevSearch;
```